src/v8/isolate.rs

```diff
@@ -21,6 +21,7 @@ use std::os::raw::{c_char, c_int};
 /// An isolate rust wrapper object.
 /// The isolate will not be automatically freed.
 /// In order to free an isolate, one must call [`V8Isolate::free_isolate`].
+#[derive(Debug, Clone, Eq, PartialEq, Ord, PartialOrd, Hash)]
```

Comment:
I do not think we can really clone this object right? Also why do we need the others?